src/platform/silabs/efr32/PlatformManagerImpl.cpp

```diff
@@ -159,5 +160,16 @@ void PlatformManagerImpl::HandleWFXSystemEvent(wfx_event_base_t eventBase, sl_wf
 }
 #endif
 
+namespace Silabs {
+
+void MigrateCounterConfigs(void)
```

Comment:
This should probably no be here and not duplicated for efr32 and 917

Reply:
Where would you reccomend this call be placed ? Not sure about a great spot for it..

Reply:
I have moved this method to MigrationManager.cpp